internal/pkg/agent/configuration/settings.go

```diff
@@ -14,13 +14,14 @@ import (
 
 // SettingsConfig is an collection of agent settings configuration.
 type SettingsConfig struct {
-	ID               string                          `yaml:"id" config:"id" json:"id"`
-	DownloadConfig   *artifact.Config                `yaml:"download" config:"download" json:"download"`
-	ProcessConfig    *process.Config                 `yaml:"process" config:"process" json:"process"`
-	GRPC             *GRPCConfig                     `yaml:"grpc" config:"grpc" json:"grpc"`
-	MonitoringConfig *monitoringCfg.MonitoringConfig `yaml:"monitoring" config:"monitoring" json:"monitoring"`
-	LoggingConfig    *logger.Config                  `yaml:"logging,omitempty" config:"logging,omitempty" json:"logging,omitempty"`
-	Upgrade          *UpgradeConfig                  `yaml:"upgrade" config:"upgrade" json:"upgrade"`
+	ID                 string                          `yaml:"id" config:"id" json:"id"`
+	DownloadConfig     *artifact.Config                `yaml:"download" config:"download" json:"download"`
+	ProcessConfig      *process.Config                 `yaml:"process" config:"process" json:"process"`
+	GRPC               *GRPCConfig                     `yaml:"grpc" config:"grpc" json:"grpc"`
+	MonitoringConfig   *monitoringCfg.MonitoringConfig `yaml:"monitoring" config:"monitoring" json:"monitoring"`
+	LoggingConfig      *logger.Config                  `yaml:"logging,omitempty" config:"logging,omitempty" json:"logging,omitempty"`
+	EventLoggingConfig *logger.Config                  `yaml:"logging.event_data,omitempty" config:"logging.event_data,omitempty" json:"logging.event_data,omitempty"`
```

Comment:
Because those logs stay on `data/elastic-agent-8.13.2-6d53d8/logs/` and the configuration for this log output is hardcoded and not documented I followed the same approach and kept the event logging configuration hardcoded and not documented.

My goal was to keep it consistent with the other logs that are collected by the monitoring Filebeat.

I can make it configurable and document, but I believe it will be confusing to only have part of the logs from `data/elastic-agent-8.13.2-6d53d8/logs/` being configurable.

Reply:
Ah yes I keep forgetting we log twice.